nix/sources.json

```diff
@@ -11,6 +11,18 @@
         "url": "https://github.com/agda/agda-stdlib/archive/v1.0.1.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "all-hies": {
+        "branch": "master",
+        "description": "Cached Haskell IDE Engine Nix builds for all GHC versions",
+        "homepage": "",
+        "owner": "reactormonk",
```

Comment:
Have you reported the issues that mean we need a fork upstream?

Reply:
https://github.com/reactormonk/all-hies/commit/1b2ecfcb0e5c058bfe0a4188b7ea4a847f9a2b92

The update comes once it's finished (takes a few hours -.-)

Reply:
And update: https://github.com/Infinisil/all-hies/pull/36

Reply:
Great, I'm happy so long as you're keeping an eye on it.